php81/php_zip.c

```diff
@@ -1156,10 +1156,15 @@ static PHP_MSHUTDOWN_FUNCTION(zip);
 static PHP_MINFO_FUNCTION(zip);
 /* }}} */
 
+static const zend_module_dep zip_deps[] = {
+	ZEND_MOD_REQUIRED("pcre")
+	ZEND_MOD_END
+};
+
 /* {{{ zip_module_entry */
 zend_module_entry zip_module_entry = {
-	STANDARD_MODULE_HEADER,
-	"zip",
```

Comment:
This one should not be removed ;)

Reply:
Ah, indeed. Thanks. I've pasted it too quickly.